en/telemetry/esp32_wifi_module.md

```diff
@@ -1,169 +1,159 @@
+| 3                 | WiFi Access Point Mode LR      | WPA2 PSK       | ESP32 launches WiFi access point mode using espressifs LR mode                                                                                           | Only ESP32 LR Mode enabled devices can detect and connect to the access point. Data rate is reduced to 0.25Mbit. Range is greatly increased.                              |
+| 4                 | ESP-NOW LR Mode AIR            | AES256-GCM     | ESP32 is able to receive ESP-NOW broadcast packets from any GCS in the area and forwards them to the UART. Broadcasts to all GND stations in the area.   | Connectionless protocol. Data rate is reduced to 0.25Mbit. Range is greatly increased compared to WiFi modes. Custom encryption mode for ESP-NOW broadcasts and protocol. |
+| 5                 | ESP-NOW LR Mode GND            | AES256-GCM     | ESP32 is able to receive ESP-NOW broadcast packets from any drone in the area and forwards them to the UART. Broadcasts to all AIR stations in the area. | Connectionless protocol. Data rate is reduced to 0.25Mbit. Range greatly increased compared to WiFi modes. Custom encryption mode for ESP-NOW broadcasts and protocol.    |
 
-![DroneBridge for ESP32 Webinterface](../../assets/peripherals/telemetry/esp32/dbesp32_webinterface.png)
+## Configuring the Ground Control Station
 
-:::tip
-Some settings require you to reboot the ESP32 to take effect.
-:::
+QGroundControl should auto-detect the connection and no further actions should be necessary (_DroneBridge for ESP32_ automatically forwards data from all connected WiFi devices via UDP to port 14550).
 
-### API
+The following connection options are available:
+
+- UDP unicast on port `14550` to all connected devices.
+- TCP on port `5760`
+
+### API & Troubleshooting
 
 DroneBridge for ESP32 offers a REST:API that allows you to read and write configuration options.
-You are not limited to the options presented by the Webinterface (e.g. baud rates).
-You can use the API to set custom baud rates or to integrate the system into your own setup.
-
-**To request the settings**
-
-```http request
-http://dronebridge.local/api/settings/request
-```
-
-**To request stats**
-
-```http request
-http://dronebridge.local/api/system/stats
-```
-
-**Trigger a reboot**
-
-```http request
-http://dronebridge.local/api/system/reboot
-```
-
-**Trigger a settings change:** Send a valid JSON
-
-```json
-{
-  "wifi_ssid": "DroneBridge ESP32",
-  "wifi_pass": "dronebridge",
-  "ap_channel": 6,
-  "tx_pin": 17,
-  "rx_pin": 16,
-  "telem_proto": 4,
-  "baud": 115200,
-  "msp_ltm_port": 0,
-  "ltm_pp": 2,
-  "trans_pack_size": 64,
-  "ap_ip": "192.168.2.1"
-}
-```
-
-to
-
-```http request
-http://dronebridge.local/api/settings/change
-```
-
-## Troubleshooting
-
-- Always erase the flash of the ESP32 before flashing a new release/firmware
```

Comment:
Are any of these still "always do" instructions? Specifically should you always erase the old firmware? Is there a link to specific trouble shooting instructions?

Reply:
Added link to trouble shooting page